src/stuf/stuf_bg/client.py

```diff
@@ -42,15 +43,24 @@ def _get_request_base_context(self):
         }
 
     def _make_request(self, data):
+        url = self.service.get_endpoint(type=EndpointType.vrije_berichten)
+
+        stuf_bg_request(self.service, url, data)
 
         response = requests.post(
-            self.service.get_endpoint(type=EndpointType.vrije_berichten),
+            url,
             data=data,
-            headers={"Content-Type": "text/xml"},
+            headers={
+                "Content-Type": SOAP_VERSION_CONTENT_TYPES.get(
+                    self.service.soap_version
+                )
+            },
             cert=self.service.get_cert(),
             auth=self.service.get_auth(),
         )
 
+        stuf_bg_response(self.service, url, str(response.content))
```

Comment:
checked with Joeri - do NOT log this sort of data in the database. However, it is okay to add it to the python logging (`logger.info(...)`) so that we can debug better. The Docker daemon/container configuration is configured to rotate the logs automatically so that any sensitive data disappears over time